CONTRIBUTING.md

```diff
@@ -46,7 +52,7 @@ sbt
 # Inside a new terminal window
 cd vscode-extension
 npm install
-code vscode-extension
+code .
```

Comment:
👍

Reply:
VSCode-specific part should mention that you need to install some basic Scala extension first, without it Metals extension won't get activated.

Reply:
Done.

Reply:
That may explain some setup issues I've had with some colleagues! Very nice catch, @laughedelic!